### PR TITLE
Fix Readme Link

### LINKED ISSUE #1: First README link broken
Currently the link points to -> https://o0ignition0o.github.io/rust-lang-blog-boilerplate/
but I don't think it's hosted there anymore and is instead here -> https://blog.jeremylempereur.com/
but was too unsure to actually open a PR to fix it.

### PR DESCRIPTION
**Pull request by callym**
This fixes #1 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-on [GitHub pages].
+This is the boilerplate code that allows you to quickly setup a blog that looks like the [RustLang blog]. It is running
+It can easily be run on [GitHub pages].
 
-[the rustlang blog boilerplate]: https://o0ignition0o.github.io/rust-lang-blog-boilerplate/
-[GitHub pages]: https://pages.github.com/
+This project has been forked from the awesome [github repo](https://github.com/rust-lang/blog.rust-lang.org) of the [RustLang blog].
 
-This project has been forked from the awesome [github repo](https://github.com/rust-lang/blog.rust-lang.org) of the [RustLang blog](https://blog.rust-lang.org/)
+[RustLang blog]: https://blog.rust-lang.org/
```
Comment (o0Ignition0o) [inline README.md]: I like the refactoring, maybe we could do it on line 6 too ? :)

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The Rust Programming Language Blog is primarily distributed under the terms of
 CC-BY 4.0. 
 So is this boilerplate.
 
-See LICENSE for details.
+See [LICENSE](LICENSE) for details.
```
Comment (o0Ignition0o) [inline README.md:34]: Nice catch, I like it !

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # RustLang Blog Boilerplate
 
-This is the code that powers [the rustlang blog boilerplate]. It is running
-on [GitHub pages].
+This is the boilerplate code that allows you to quickly setup a blog that looks like the [RustLang blog]. It is running
```
Comment (o0Ignition0o) [inline README.md]: Can we remove the "It is running" statement ? :)

**Comment by o0Ignition0o** [comment]
This is a really nice pull request, thanks a lot for submitting it :)

A couple of tiny nits to fix and this will be ready to merge 🥇

File: README.md (hunk 3 above)
Comment (callym) [reply README.md]: Oops! Glad you caught that!

**Comment by callym** [comment]
Thanks! I've changed the things you flagged up 😄

**Comment by o0Ignition0o** [comment]
Wow thanks for doing such a great job! 
You're now the first awesome code contributor to this project !
If you would like to contribute some more, don't hesitate to let me know !

**Pull request merged**